valhalla/loki/reach.h

```diff
@@ -60,13 +60,18 @@ class Reach : public thor::Dijkstras {
   // edges which the costing could decide to skip (because of restrictions and possibly more?)
   // when that happens and the maximum reach is not found, this is then validated with a more
   // accurate exact expansion performed by the method below
-  directed_reach exact(const valhalla::baldr::DirectedEdge* edge,
+  directed_reach exact(const baldr::DirectedEdge* edge,
                        const baldr::GraphId edge_id,
                        uint32_t max_reach,
-                       valhalla::baldr::GraphReader& reader,
+                       baldr::GraphReader& reader,
                        const std::shared_ptr<sif::DynamicCost>& costing,
                        uint8_t direction = kInbound | kOutbound);
 
+  void enqueue(const baldr::GraphId& node_id,
```

Comment:
maybe also copy the comment here to describe what this method does. probably this and the clear method and the member variables should be marked `protected` as well